src/Module/Post/Edit.php

```diff
@@ -182,6 +182,11 @@ protected function content(array $request = []): string
 			'$shortpermset' => $this->t('Permissions'),
 
 			'$compose_link_title' => $this->t('Open Compose page'),
+			// Dropzone
+			//'$max_imagesize'       => DI::config()->get('system', 'maximagesize'),
+			// DI::config.. does not work here, so it is set to a manual value
```

Comment:
`DI::config()->get('system', 'maximagesize')` should work here.

Reply:
I don't know why... but when i use it, i geht a http 500 when i try to edit posting or comment.

Reply:
"Not working" means... the edit-overlay does not appear. In Devtools i see the Request with answer "500 Internal Server Error"